index.html

```diff
@@ -23,18 +23,129 @@
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="card-tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="card-btn" type="submit">See Project</button>
+            </div>
+        </section>
+        <section class="cards">
+            <img src="../src/assets/images/Snapshoot_Portfolio_1.svg" alt="img">
+            <div class="work-block">
+                <h2>Multi-Post Stories</h2>
+                <ul class="list-items">
+                    <li class="count">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="card-tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="card-btn" type="submit">See Project</button>
+            </div>
+        </section>
+        <section class="cards">
+            <img src="../src/assets/images/Snapshoot_Portfolio_2.svg" alt="img">
+            <div class="work-block">
+                <h2>Tonic</h2>
+                <ul class="list-items">
+                    <li class="count">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="card-tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="card-btn" type="submit">See Project</button>
+            </div>
+        </section>
+        <section class="cards">
+            <img src="../src/assets/images/Snapshoot_Portfolio_3.svg" alt="img">
+            <div class="work-block">
+                <h2>Multi-Post Stories</h2>
+                <ul class="list-items">
+                    <li class="count">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="card-tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="card-btn" type="submit">See Project</button>
+            </div>
+        </section>
+        <section class="cards">
+            <h1>About Myself</h1>
+            <p>I’m a software developer! I can help you build a product ,
```

Comment:
- [OPTIONAL] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> 💡 You used the h1 tag once here `<h1>I am TEEJAY! Happy to have you here today!!</h1>` and that's great because this is the main title 👌